src/views/Admin/ReportingTool.vue

```diff
@@ -200,6 +200,24 @@
         </div>
       </div>
     </div>
+    <div class="container-fluid" v-show="productPieData">
+      <div class="card">
+        <h3 class="mx-5 my-5">Order By Day Statistics</h3>
+        <div class="card-body">
+          <div class="row">
+            <div class="col-md-12">
+              <template v-if="reports !== null">
```

Comment:
why not  `v-if="!reports"`

Reply:
isn't this for negating values that are boolean ?

Reply:
Not only booleans